src/components/Profile.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+  import { storeMsaInfo } from '$lib/stores';
+  import type { MsaInfo } from '$lib/storeTypes';
+
+  let msaInfo: MsaInfo;
+
+  storeMsaInfo.subscribe((info) => {
```

Comment:
I think you can auto subscribe like
https://learn.svelte.dev/tutorial/auto-subscriptions